ADDING-A-THEME.md

```diff
@@ -156,6 +156,67 @@ of the tree.
 If you don't have GNU make (ex. on Windows), don't worry about doing
 this step, it's entirely trivial for me to do it.
 
+IMPORTANT NOTE 2023-05-11 by Evgueni Antonov (github: StrayFeral): 
```

Comment:
This file is meant as instructions on how to add new themes.

It's not really the right place to write about your experiences :).

Also, we should not encourage people to continue using Python2 but rather port the code to Python3.

I would say just remove the changes in this file.